assets/templates/localmetrics/prometheus-rule.yaml

```diff
@@ -27,3 +27,13 @@ spec:
             A frequent issue is that nodes that run OpenShift Data Foundation (ODF) are tainted,
             so only ODF runs on these nodes. In that case, LocalVolume / LocalVolumeSet objects
             must tolerate such a taint to provide disks for ODF on the nodes.
+    - name: lvset_deletion_alerts
+      rules:
+      - alert: LvSetDeletionAlert
+        expr: lso_lvset_deletion_timestamp > 0 and time() - lso_lvset_deletion_timestamp > 72 * 3600
+        for: 1m
+        labels:
+          severity: warning
+        annotations:
+          summary: "LocalVolumeSet has had a deletion timestamp older than 72 hours"
+          description: "LocalVolumeSet {{ $labels.lvSetName }} has been marked for deletion for more than 72 hours."
```

Comment:
How did you test this? keeping the metric being emitted for 72 hours?

Reply:
I deployed an operator with a slightly modified version of this patch, replacing `72 * 3600` in line 33  with 60 seconds.

Reply:
So if we want to use a shorter time interval, then that changes things. such as - we can just emit a gauge with values 0 or 1 and then if value is `>0` for  "30m" then, we emit the alert. We no longer have to do subtraction and mess with timestamps.

Reply:
I guess that's another way of doing it, but I think that way we'll just move the timestamp substraction into the metric logic (code) instead of keeping it in the alert?

I could be wrong, but to me it seems cleaner to make the interval (72h, 30m, etc.) configurable in the alert instead?